src/remote-config/remote-config.ts

```diff
@@ -89,7 +132,7 @@ export class RemoteConfigTemplate {
    * @return {RemoteConfigParameter} The Remote Config parameter with the provided key.
    */
   public getParameter(key: string): RemoteConfigParameter | undefined {
-    return this.parameters.find((p) => p.key === key);
+    return this.parameters[key];
```

Comment:
I think we no longer need this method. Developers can always do `template.parameters[key]`